src/components/EditCoursePage/EditCoursePage.test.jsx

```diff
@@ -10,14 +10,15 @@ import StatusAlert from '../StatusAlert';
 
 import { PUBLISHED, REVIEW_BY_INTERNAL, REVIEW_BY_LEGAL, UNPUBLISHED } from '../../data/constants';
 import { courseOptions, courseRunOptions } from '../../data/constants/testData';
+import { jsonDeepCopy } from '../../utils';
 
 // Need to mock the Editor as we don't want to test TinyMCE
 jest.mock('@tinymce/tinymce-react');
 
 const mockStore = configureStore();
 
 describe('EditCoursePage', () => {
-  const defaultPrice = '77.00';
```

Comment:
Is this because of the formatting you added in? Assuming `77.01` would have still be fine

Reply:
Yes, 77.01 would have been fine too. I was just chopping things up.